Source/WTF/wtf/FixedVector.h

```diff
@@ -236,25 +237,25 @@ size_t FixedVector<T>::findIf(const MatchFunction& matches) const
     return notFound;
 }
 
-template<typename T>
+template<typename T, typename Malloc>
 template<typename U>
-size_t FixedVector<T>::find(const U& value) const
+size_t FixedVector<T, Malloc>::find(const U& value) const
 {
     return findIf([&](auto& item) {
         return item == value;
     });
 }
 
-template<typename T>
-inline void swap(FixedVector<T>& a, FixedVector<T>& b)
+template<typename T, typename Malloc>
+inline void swap(FixedVector<T, Malloc>& a, FixedVector<T, Malloc>& b)
 {
     a.swap(b);
 }
 
-template<typename T, typename MapFunction, typename ReturnType = typename std::invoke_result<MapFunction, const T&>::type>
-FixedVector<ReturnType> map(const FixedVector<T>& source, MapFunction&& mapFunction)
+template<typename T, typename MapFunction, typename ReturnType = typename std::invoke_result<MapFunction, const T&>::type, typename Malloc = EmbeddedFixedVectorMalloc>
```

Comment:
Is this default really necessary?  Would the compiler just infer it from the `const FixedVector<T, Malloc>& source` argument?

Reply:
It's necessary since the prior template argument has a default, but it seems like nobody is using this right now so I can move the Malloc function to the third place so that it's before any defaulted template args.